src/script/Cargo.toml

```
@@ -5,7 +5,7 @@ version.workspace = true
 license.workspace = true
 
 [features]
-default = ["python"]
+default = ["python", "pyo3"]
```

```suggestion
default = ["python"]
```

Looks like this will require the develop env to have `python3` ([source](https://github.com/pyo3/pyo3#using-python-from-rust)). Prefer not to default open this feature gate to alleviate the developer's burden (though `python3` is very common...). We can manually open it when testing related code (e.g., in CI) via `cargo test -F pyo3`.

And by the way, we should also state this requirement in our developer docs (and update readme correspondingly).